### PR TITLE
Replace repetitive actions with Action Groups in AdminCheckInactiveAndNotIncludeInMenuCategoryAndSubcategoryIsNotVisibleInNavigationTest and AdminCheckInactiveCategoryAndSubcategoryIsNotVisibleInNavigationMenuTest

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Fixed Issues (if relevant)
Test is refactored according to the best practices followed by MFTF.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Review by sidolov — CHANGES_REQUESTED** [review]
@kate-kyzyma could you please take a look on SVC failure? Remove node can be replaced with a comment with the same stepKey

**Comment by kate-kyzyma** [comment]
@magento run Semantic Version Checker

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminCheckInactiveAndNotIncludeInMenuCategoryAndSubcategoryIsNotVisibleInNavigationTest.xml [hunk 1]
```diff
@@ -33,18 +33,26 @@
         <actionGroup ref="AdminOpenCategoryPageActionGroup" stepKey="openAdminCategoryIndexPage"/>
         <!--Create subcategory under parent category -->
         <actionGroup ref="AdminExpandCategoryTreeActionGroup" stepKey="clickOnExpandTree"/>
-        <click selector="{{AdminCategorySidebarTreeSection.categoryInTree($$createCategory.name$$)}}" stepKey="selectCategory"/>
-        <waitForPageLoad stepKey="waitForPageToLoad"/>
+        <actionGroup ref="AdminCategoriesOpenCategoryActionGroup" stepKey="openCategory">
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/Test/AdminCheckInactiveAndNotIncludeInMenuCategoryAndSubcategoryIsNotVisibleInNavigationTest.xml]: Please use the same stepKey
```suggestion
        <actionGroup ref="AdminCategoriesOpenCategoryActionGroup" stepKey="selectCategory">
```

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by kate-kyzyma** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by eduard13** [comment]
@magento run Functional Tests B2B, Integration Tests

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by kate-kyzyma** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Review by eduard13 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-9337](https://jira.corp.magento.com/browse/ENGCOM-9337) has been created to process this Pull Request

**Pull request merged**